Blog post about new features and FAQ update (#10)
rero/sonar.ch: content/post/state-of-the-new-features-by-the-end-of-2021.md

---
title: "State of the new features by the end of 2021"
date: 2021-11-30T11:00:00+02:00
draft: false
tags: ["features", "subdivisions", "swisscovery", "sorting", "multilingual"]
---

With the go-live of the first SONAR institutions, the system has quickly evolved to complete its core functionalities. Here is a brief overview of the main one implemented since this summer.

<!--more-->

## Subdivisions

A subdivision enables to group users (submitters and moderators) within the organisation, for example according to their department, institute or faculty. This makes it possible to manage rights and organise workflows more finely in the repository.

> For example, a moderator within a subdivision is notified only of new submissions from researchers of his·her own subdivision. He or she can see, edit and delete these deposits, but not those of other subdivisions. In this way, the validation work can be shared and organised within the institution, especially in big universities.

Reviewer: ```suggestion
For example, a moderator within a subdivision is notified only of new submissions from researchers of his·her own subdivision. He or she can see, edit and delete these deposits, but not those of other subdivisions. In this way, the validation work can be shared and organised within the institution, especially in larger universities.
```

See more details in the [SONAR help](https://sonar.ch/help/subdivisions/).

## Import from swisscovery

> During the deposit process, it is possible to import the metadata from an existing record in swisscovery. thus, the data does not have to be entered manually twice.

Reviewer: ```suggestion
During the deposit process, it is possible to import the publication's metadata from an existing record in swisscovery, thus avoiding to enter it manually.
```

## Sorting results

The documents search view is equipped with sorting options by title, date and relevance, the latter being applied by default. But also the other search views in the professional interface have these options, for instance sorting deposits by date or users by name.

## Multilingual configuration of the organisation homepage and footer

The administrator of the organisation can configure and update the homepage texts in DE/EN/FR/IT without specific programming or web editing knowledge. The text can be easily formatted (see below).

{{< figure src="/images/organisation_editor.jpg" caption="Organisation editor for more multilingualism" alt="Organisation editor for more multilingualism" width="100%" >}}

## Button for submitting publications

An easy-to-see button for depositing publications is present in the header of SONAR. It appears only for connected submitters/researchers. Thus, it is visible from any page of the interface and encourage researchers to deposit their papers.

## Document visibility settings

> [In this blog post](/post/autumn-tests-and-new-features/#new-features), we already told you about managing file accesses (open, embargoed, restricted access), independently of the visibility of the bibliographic metadata. Now it is also possible to act on the metadata, for example to mask a document in the public interface, or to show it only for users within the IP addresses of the organisation.

Reviewer: ```suggestion
[In a recent blog post](/post/autumn-tests-and-new-features/#new-features), we already told you about managing file access rights (open, embargoed, restricted access), independently of the visibility of the bibliographic metadata. Now it is also possible to act on the metadata, for example to mask a document in the public interface, or to show it only for users within the IP addresses of the organisation.
```
